exporters/scss/exporter.json

```diff
@@ -5,7 +5,7 @@
   "author": "Jan Kryšpín <jan.kryspin@lmc.eu>",
   "organization": "LMC s.r.o.",
   "source_dir": "src",
-  "version": "2.3.1",
+  "version": "2.4.1",
```

Comment:
Shouldn't be version 2.4.0?

Reply:
Sure, 
![image](https://user-images.githubusercontent.com/7041807/222428293-91ff07d9-2e87-4497-9b24-dcae70f5b14a.png)
sorry